html/pages/services.inc.php

```diff
@@ -131,8 +131,8 @@
         <td><?php echo $devlink?></td>
         <td><?php echo $status?></td>
         <td><?php echo formatUptime(time() - $service['service_changed'])?></td>
-        <td><span class='box-desc'><?php echo nl2br(trim($service['service_message']))?></span></td>
-        <td><span class='box-desc'><?php echo nl2br(trim($service['service_desc']))?></span></td>
+        <td><span class='box-desc'><?php echo nl2br(trim(stripslashes($service['service_message'])))?></span></td>
```

Comment:
Because it doesn't need to escape special characters.  For some reason I thought this line was the same as 135, but apparently they aren't now...